addons/full-import/import.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+set -o nounset -o pipefail -o errexit
+
+source /usr/local/pf/addons/full-import/helpers.functions
+source /usr/local/pf/addons/full-import/database.functions
+source /usr/local/pf/addons/full-import/configuration.functions
+
+dump_path="$1"
+
+if ! [ -f "$dump_path" ]; then
+  echo "Please specify a valid archive to extract"
+  echo "Usage: import.sh /path/to/export.tgz"
+  exit 1
+fi
+
+extract_dir=`mktemp -d`
+
+cp -a $dump_path $extract_dir/export.tgz
+check_code $?
+
+pushd $extract_dir/
+check_code $?
+
+main_splitter
+
+echo "Extracting archive..."
+tar -xf export.tgz
+check_code $?
+
+echo "Found the following content in the archive:"
+ls -l
+
+main_splitter
+files_dump=`ls packetfence-files-*`
+echo "Found files dump '$files_dump'"
+
+echo "Extracting files dump"
+tar -xf $files_dump
+check_code $?
+
+main_splitter
+
+db_dump=`ls packetfence-db-dump-*`
+echo "Found compressed database dump '$db_dump'"
+gunzip $db_dump
+check_code $?
+db_dump=`ls packetfence-db-dump-*`
+echo "Uncompressed database dump '$db_dump'"
+
+main_splitter
+echo "Stopping PacketFence services"
+systemctl cat monit >/dev/null 2>&1 && (systemctl stop monit ; systemctl disable monit)
+systemctl isolate packetfence-base
+check_code $?
+systemctl stop packetfence-galera-autofix
```

Comment:
Is it necessary to stop it here because we say in README that we don't support restoring on a cluster ?

Reply:
I'm scared that someone would do it on a cluster they started building and then galera autofix starts doing stuff with the other nodes. Unlikely but its a safety net

Reply:
I understand, let's keep it.